tools/mason/MasonPublish.chpl

```diff
@@ -443,8 +446,8 @@ private proc addPackageToBricks(projectLocal: string, safeDir: string, name : st
         baseToml["brick"]!.set("source", projectLocal);
         tomlWriter.write(baseToml);
         tomlWriter.close();
-        const gitMessageString = ('git tag -a v' + versionNum + ' -m  "' + name + '"');
-        gitC(projectLocal, gitMessageString);
+        const gitMessageString = ('git tag -a "v' + versionNum + '" -m "' + name + '"');
+        commitSubProcess(projectLocal, ['git','tag','-a',"v"+versionNum,'-m',name]);
```

Comment:
`commitSubProcess` here runs a `git tag` inside of the local-registry. Please check if its the correct behavior.

Reply:
I believe so. Is this responsible for one of the errors above?

Reply:
Yes, it is.